sdk/core/core/inc/az_http_policy.h

```diff
@@ -58,6 +59,16 @@ AZ_NODISCARD az_result az_http_pipeline_policy_authentication(
     az_http_request_builder * const hrb,
     az_mut_span const * const response);
 
+AZ_INLINE AZ_NODISCARD az_result
+az_http_pipeline_policy_authentication_with_secret_credential_build(
+    az_client_secret_credential * credential,
+    az_http_policy * out) {
+  AZ_CONTRACT_ARG_NOT_NULL(out);
+  *out = (az_http_policy){ .pfnc_process = az_http_pipeline_policy_authentication,
+                           .data = credential };
```

Comment:
add after the last element, it will format the initialization without horizontal alignment. (`.data = credential,`)